pipelines/templates/tasks/unitybuild.yml

```diff
@@ -7,11 +7,19 @@ parameters:
   ScriptingBackend: 'default'  # [optional] default|.NET
   PublishArtifacts: false
   PackagingDir: 'Win32'
+  UnityVer: 'Unity2018'

```

Comment:
Why not pass the actual path in here? I.e. make it default to {$Env:${Unity2019Version)} and pass in the path to the editor if different.

Reply:
Sounds good to me.